dapps/marketplace/src/utils/profileTools.js

```diff
@@ -1,69 +1,77 @@
 import get from 'lodash/get'
+import { fbt } from 'fbt-runtime'
 
 const websiteAttestationEnabled =
   process.env.ENABLE_WEBSITE_ATTESTATION === 'true'
 
 export function unpublishedStrength({ props, state }) {
   // TODO: Retrieve stregths from GraphQL?
   const profile = get(props, 'identity') || {}
-  const verifiedAttestations = mapVerifiedAttestations(profile)
+  const allProviders = props.attestationProviders
+  const verifiedAttestations = (profile.verifiedAttestations || []).map(
+    att => att.id
+  )
+
+  let strength = allProviders.reduce((sum, provider) => {
+    if (
+      verifiedAttestations.includes(provider) ||
+      !state[`${provider}Attestation`]
+    ) {
+      return sum
+    }
+
+    switch (provider) {
+      case 'email':
+      case 'phone':
+      case 'facebook':
+      case 'google':
+      case 'twitter':
+        sum += 10
```

Comment:
Lets just add 10% for each attestation but max it out at 100.

Reply:
What about first name, last name, avatar and description? Same 10?

Reply:
Sure that's fine for now